source/documentation/managing_apps/quotas.md

```diff
@@ -117,3 +117,17 @@ The application cannot access more than the specified amount of memory.
 
 		... description
 		... index: 0, reason: CRASHED, exit_description: out of memory, exit_status: 255
+
+
+### Quotas (as of 2021-07-28)
+
+| Name            | Memory (MB) | Routes | Services |
```

Comment:
@markpeterbuckley I added the unit clarification to Memory, if that looks OK to you?

Reply:
Needs to match the titles in the bullets above (or vice versa)
So Memory (MB) > RAM
Services > Service instances